server/tilesource/__init__.py

```diff
@@ -20,21 +20,42 @@
 # Deal with a bug where PEP257 crashes when parsing __all__
 # flake8: noqa
```

Comment:
I wonder if PEP257 will still crash now that we build `__all__` in this way. I'll file a bug to explore it.